packages/usa-tooltip/src/styles/_usa-tooltip.scss

```diff
@@ -66,6 +70,15 @@ $triangle-size: 5px;
   opacity: 1;
 }
 
+.usa-tooltip__body--top {
+  &:before {
+    height: $triangle-size;
+    left: 0;
+    right: 0;
+    top: 100%;
+  }
```

Comment:
I tried this out and it it didn't seem to work for left and right placement. The mixin for this would need to be a bit more involved, so I am torn on the benefit here. I lean towards leaving the code as is. @mejiaj do you have a preference?

Reply:
A mixin would help improve readability. Currently there's no sign of what this pseudoelement is for, but it could be an enhancement. Unless you want to give it another shot.

Reply:
I added a comment in  [39534b5](https://github.com/uswds/uswds/pull/5919/commits/39534b5c64d0768f7a92f77249c68df7c3b347ab) to help improve clarity. If you think the comment is insufficient, I can work on the mixin on Monday.

Reply:
Update: @mejiaj created a mixin for this in #5933 (Thanks James!). It was merged into this PR in commit https://github.com/uswds/uswds/pull/5919/commits/babcc2a3e74e22b7793325bd07db512f79efadc1. Going to resolve this comment thread since the task is complete.